web/web.go

```diff
@@ -236,10 +237,22 @@ func (h *Handler) Reload() <-chan struct{} {
 	return h.reloadCh
 }
 
+type errorLogWriter struct{}
+
+func (h *errorLogWriter) Write(b []byte) (int, error) {
```

Comment:
You'll also want to change this from `h` to something else now (since it's no longer called `hub`). You could actually omit it, because it's unused. Also, it doesn't need to be a pointer receiver, so how about just: `func (errorLogWriter) Write(b []byte) (int, error) {`